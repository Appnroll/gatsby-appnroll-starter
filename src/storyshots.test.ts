/**
 * This file initializes the StoryShots functionality.
 * Read more: https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core
 */

import initStoryshots from "@storybook/addon-storyshots"
import "jest-styled-components"

initStoryshots({
  /* configuration options */
})
