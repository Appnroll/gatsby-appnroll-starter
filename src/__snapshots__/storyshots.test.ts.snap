// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Base|Footer headers 1`] = `
<footer>
  Â© 
  2077
  , Made by
   
  <a
    href="https://www.appnroll.com/"
    title="App'n'roll"
  >
    <test-file-stub />
  </a>
</footer>
`;

exports[`Storyshots Base|Header headers 1`] = `
.c0 {
  background: hotpink;
  margin-bottom: 1.45rem;
}

.c1 {
  margin: 0 auto;
  max-width: 960px;
  padding: 1.45rem 1.0875rem;
}

<div
  style={
    Object {
      "boxShadow": "rgba(102, 119, 136, 0.03) 0px 6px 8px, rgba(102, 119, 136, 0.3) 0px 1px 2px",
      "maxHeight": "70vh",
      "maxWidth": "70vw",
      "overflow": "auto",
      "padding": "30px",
    }
  }
>
  <header
    className="c0"
  >
    <div
      className="c1"
    >
      <h1>
        Title
      </h1>
    </div>
  </header>
</div>
`;
